app/controllers/homepage_controller.rb

```diff
@@ -297,6 +298,26 @@ def search_coordinates(latlng)
     end
   end
 
+  def bounding_box_center(corners)
+    lat1, lng1, lat2, lng2 = corners.split(',').map { |n| toRadians(n) }
+    bx = Math.cos(lat2) * Math.cos(lng2 - lng1)
+    by = Math.cos(lat2) * Math.sin(lng2 - lng1)
+    lat_c = Math.atan2(Math.sin(lat1) + Math.sin(lat2),
+                        Math.sqrt((Math.cos(lat1)+bx)*(Math.cos(lat1)+bx)+by*by))
+    lng_c = lat1 + Math.atan2(by, Math.cos(lng1) + bx)
+    { latitude: toDegrees(lat_c), longitude: toDegrees(lng_c) }
```

Comment:
@mporkola, yesterday, I tried to [link] (http://stackoverflow.com/questions/29322582/how-to-calculate-center-of-bounds-without-googlemaps) you this solution. Sorry, I wasn't more explicit.
```
var center = (new google.maps.LatLngBounds(bounds.southeast, bounds.northeast)).getCenter();
```